langchain/memory/chat_memory.py

```diff
@@ -1,14 +1,21 @@
 from abc import ABC
 from typing import Any, Dict, List, Optional
 
-from pydantic import BaseModel, Field
+from pydantic import Field
 
 from langchain.memory.utils import get_prompt_input_key
-from langchain.schema import AIMessage, BaseMemory, BaseMessage, HumanMessage
+from langchain.schema import (
+    AIMessage,
+    BaseChatMessageHistory,
+    BaseMemory,
+    BaseMessage,
+    HumanMessage,
+)
 
 
-class ChatMessageHistory(BaseModel):
-    messages: List[BaseMessage] = Field(default_factory=list)
+class ChatMessageHistory(BaseChatMessageHistory):
+    messages: List[BaseMessage] = []
+    session_id: str = "default"
```

Comment:
i dont think this is needed here?

Reply:
It should probably still inherit from `BaseChatMessageHistory`, but yes, the `session_id: str = "default"` is no longer required. That was only there as in an earlier version `BaseChatMessageHistory` had the `session_id` as a mandatory attribute.